lms/djangoapps/program_enrollments/models.py

```diff
@@ -162,3 +165,25 @@ def enroll(cls, program_enrollment, course_key, status):
             status=status,
         )
         return program_course_enrollment.status
+
+    def change_status(self, status):
+        """
+        Modify ProgramCourseEnrollment status and course_enrollment status if it exists
+        """
+        if status == self.status:
+            return status
+
+        self.status = status
+        if self.course_enrollment:
+            if status == CourseEnrollmentResponseStatuses.ACTIVE:
+                self.course_enrollment.activate()
+            else:
```

Comment:
You might want to explicitly check for an "inactive" status here, in case there's a different choice in the future, or if bad data ends up getting passed to this function.